### PR TITLE
Minor updates to the GitHub documentation

### PR DESCRIPTION
**Pull request by mikewheaton**
Just a few clarifications and typo fixes as I was reviewing the documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,31 +10,31 @@ Fabric is a responsive, mobile-first, collection of styles and tools designed to
...
 ##Contribute to Fabric
 
-If you have a specific bug, feature request, or question, post it via the [issue tracker](https://github.com/OfficeDev/Office-UI-Fabric/issues). For more information and labels, see our [contributing](https://github.com/OfficeDev/Office-UI-Fabric/blob/master/ghdocs/CONTRIBUTING.md) section.
+Bug reports, feature requests, and questions are posted on the [issue tracker](https://github.com/OfficeDev/Office-UI-Fabric/issues). For details on our contribution process and how we label issues, see our [Contributing](https://github.com/OfficeDev/Office-UI-Fabric/blob/master/ghdocs/CONTRIBUTING.md) page.
 
 
 ##Licenses
 
-All files on the Office UI Fabric GitHub repository are subject to the MIT license. Please read the License file at the root of the project. Usage of the fonts referenced on Office UI Fabric files is subject to the terms listed [here](http://aka.ms/fabric-font-license).
+All files on the Office UI Fabric GitHub repository are subject to the MIT license. Please read the License file at the root of the project. Usage of the fonts referenced on Office UI Fabric files is subject to the [licensehere](http://aka.ms/fabric-font-license).
```
Comment (battletoilet) [inline README.md]: Need a space between License and here

File: ghdocs/TUTORIAL.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ todo/
 |-- css/
 ```
 
-Now open up your favorite text editor or IDE and create a new file called index.html in the root directory (todo/), then add the following to it:
+Now open up your favorite text editor or IDE and create a new file called index.html in the root `todo/` directory, containg a simple HTML document:
```
Comment (battletoilet) [inline ghdocs/TUTORIAL.md]: **containg** is spelled wrong.

File: ghdocs/TUTORIAL.md [hunk 3]
```diff
@@ -166,7 +166,7 @@ Fabric includes a responsive grid system, similar to those seen in other UI tool
 </div>
 ```
 
-Now for the fun part: let's add some component HTML into our ToDo app. Navigate to where you extracted Fabric, then open the following component files in your code editor:
+Now for the fun part: let's add some components into our app! Navigate to where you extracted Fabric, then open the following component files in your code editor:
```
Comment (ericthompson) [inline ghdocs/TUTORIAL.md]: With the change, I'd probably move to "to" instead of "**into** our app."

**Comment by ericthompson** [comment]
Awesome, approved on my end!

**Pull request merged**